frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -126,20 +126,38 @@ public ModelArchive registerModel(
         return archive;
     }
 
-    public boolean unregisterModel(String modelName) {
+    public HttpResponseStatus unregisterModel(String modelName) {
         Model model = models.remove(modelName);
         if (model == null) {
             logger.warn("Model not found: " + modelName);
-            return false;
+            return HttpResponseStatus.NOT_FOUND;
         }
-
+        logger.warn("setMinWorkers(0)");
```

Comment:
Remove all logger.warn() messages.

Reply:
Yep, will do once I get the final commit out.